miss_islington/tasks.py

```diff
@@ -79,7 +79,7 @@ async def backport_task_asyncio(
                 await util.comment_on_pr(
                     gh,
                     issue_number,
-                    f"""{util.get_participants(created_by, merged_by)}, Something is wrong... I can't backport for now.
+                    f"""{util.get_participants(created_by, merged_by)} Sorry, Something is wrong... I can't backport for now.
                                    Please backport using [cherry_picker](https://pypi.org/project/cherry-picker/) on command line.
```

Comment:
As a too-frequent recipient of this message, the addition is more useless noise.  So is the completely non-specific 'wrong'.   So rather than merely closing this, I suggest reducing the noise and improving the suggestion. 
```suggestion
                    f"""{util.get_participants(created_by, merged_by)}, I can't backport for now.  Please try again later or
                                   backport using [cherry_picker](https://pypi.org/project/cherry-picker/) on command line.
```
Manual backport is only needed if there is a merge conflict (and I believe that is reported) or if something is severely broken and one does not want to wait for a fix.  For me, merge conflicts are extremely rare, and I am patient unless a new release is imminent, and I try to avoid merging close to deadlines.  So the solution is nearly always trying again, nearly always withing minutes, as the 'something wrong' is nearly always an intermittent timing issue or communication failure that is not miss-islington's fault.